superset/models/dashboard.py

```diff
@@ -184,6 +185,22 @@ def dashboard_link(self) -> Markup:
         title = escape(self.dashboard_title or "<empty>")
         return Markup(f'<a href="{self.url}">{title}</a>')
 
+    @property
+    def unique_value(self) -> str:
```

Comment:
Instead of `unique_value`, maybe call this something like `sha_sum`?

Reply:
No problem, but, this is actually an MD5 Exa, `unique_value` is aiming for a generic unique value digest. Maybe `unique_digest` or just `digest`, any ideas?

Reply:
I'm good with `digest`